### PR TITLE
Add failure message when updating community members

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/theme/base/client/communities/CommunitiesDisplayStrings.java [hunk 1]
```diff
@@ -10,4 +12,12 @@
...
 
     String agaveAppsNotSupportedToolTip();
+
+    String joinCommunityFailure();
+
+    String leaveCommunityFailure();
+
+    String removeCommunityAdminFailure(@PluralCount List<String> failedUsers);
+
+    String addCommunityAdminFailure(@PluralCount List<String> failedUsers);
```
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/theme/base/client/communities/CommunitiesDisplayStrings.java]: 👍

File: de-lib/src/main/java/org/iplantc/de/communities/client/presenter/ManageCommunitiesPresenterImpl.java [hunk 2]
```diff
@@ -383,6 +397,11 @@ public void onFailure(Throwable caught) {
 
             @Override
             public void onSuccess(List<UpdateMemberResult> result) {
+                List<String> failedUsers = getFailedMemberUpdateNames(result);
+
+                if (failedUsers.size() > 0) {
+                    announcer.schedule(new ErrorAnnouncementConfig(appearance.joinCommunityFailure()));
+                }
                 callback.onSuccess(null);
```
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/communities/client/presenter/ManageCommunitiesPresenterImpl.java]: Should we call error callback instead when the request fails for all the users ? 🤔

Comment (ash3rz) [reply de-lib/src/main/java/org/iplantc/de/communities/client/presenter/ManageCommunitiesPresenterImpl.java]: Ooh, really good catch - I think in all these cases, it actually makes more sense to call the `errorCallback`.  I'll refactor

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**